### DISCUSSION TITLE
Extended list of standard types

### DISCUSSION THREAD

**Discussion by DavidEnergie** [e2nIEE/pandapower#2279] [General]
Does anyone have and is willing to share a list of standard types that goes beyond the ones available in net.std_types?

I am currently modelling some less common nominal voltages (such as 30 kV) and transformer types (e.g. YNyn) and am wondering about existing work to rely on but also to help building.

Any pointers are very welcome.
Cheers

**Comment by vogt31337** [comment]
Not really since most models come with there own data. But GridCal has an interesting list:
https://github.com/SanPen/GridCal/blob/master/doc/rst_source/getting_started/data_sheets.rst

Maybe you could write a converter?
